### PR TITLE
A section to introduce job logs in web editor

### PR DESCRIPTION
**Pull request by Pratichhya**

### CODE REVIEW

**Comment by jdries** [comment]
looks good!

File: Applications/WebEditor.qmd [hunk 1]
```diff
@@ -95,4 +95,17 @@ Once the execution process is completed, the result is automatically saved local
...
 
+## Monitor the workflow
+
+When a batch job is created, openEO's logs feature can help with *monitoring and debugging* of workflows. The back-end typically uses this to dump information during data processing that may be relevant for the user (e.g. warnings, resource stats, …). Like many other processes, job logs can be visualized programmatically as well as in the web editor. 
+
+As shown in the following image, you have the option to access the job logs by clicking on the icon located next to the job ID at the bottom of the interface.
+
+![](./_images/joblist.png)
+
+When clicking on it, a window emerges next to the interface, as shown below:
```
Comment (soxofaan) [inline Applications/WebEditor.qmd:106]:
```suggestion
When clicking on it, a pane emerges on the right, as shown below:
```

**Review by soxofaan — COMMENTED** [review]
small suggestions

**Comment by Pratichhya** [comment]
> small suggestions

Thank you :smiley: 
changes are made based on suggestions.

**Comment by Pratichhya** [comment]
@soxofaan do you have any for this PR?

**Review by soxofaan — APPROVED** [review]

**Pull request merged**